demo_nodes_cpp/src/parameters/parameter_node.cpp

```diff
@@ -0,0 +1,57 @@
+// Copyright 2015 Open Source Robotics Foundation, Inc.
```

Comment:
My rational is the following: if a file was written by Alice in 2016 and in 2017 copied by Bob to a new location the copyright is clearly still lying with Alice in 2016. Bob could only add his copyright and the current year if he did some non-trivial changes.